<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>671</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>702</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QVTKOpenGLWidget" name="qvtkWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Camera_View_groupbox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>341</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>                            Modify Position of Model</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <widget class="QSpinBox" name="Shift_horizontally">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="minimum">
          <number>-999</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="stepType">
          <enum>QAbstractSpinBox::DefaultStepType</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Shift_vertically_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Shift Vertically</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="Shift_vertically">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="minimum">
          <number>-999</number>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Shift_horizontally_Label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>26</height>
          </size>
         </property>
         <property name="text">
          <string>Shift Horizontally</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="Camera_view_reset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string>Resest Camera</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="X_Plane_groupBox">
         <property name="title">
          <string>                                                  X-Axis</string>
         </property>
         <widget class="QLCDNumber" name="LCD_Number_X">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>91</width>
            <height>26</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
         </widget>
         <widget class="QDial" name="X_Plane_Position">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>121</width>
            <height>100</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="Y_Plane_groupBox">
         <property name="title">
          <string>                                                  Y-Axis</string>
         </property>
         <widget class="QDial" name="Y_Plane_Position">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>131</width>
            <height>101</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_Number_Y">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>91</width>
            <height>26</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="Z_Plane_groupBox">
         <property name="title">
          <string>                                                  Z-Axis</string>
         </property>
         <widget class="QDial" name="Z_Plane_Position">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>-10</y>
            <width>131</width>
            <height>101</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>360</number>
          </property>
          <property name="wrapping">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLCDNumber" name="LCD_Number_Z">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>60</y>
            <width>91</width>
            <height>26</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>26</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>871</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="4">
      <widget class="QGroupBox" name="LightsGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Apply Light to Model</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>151</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="0" column="0">
          <widget class="QPushButton" name="Apply_Light">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Ctrl+L</string>
           </property>
           <property name="text">
            <string>Apply Light</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+L</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="Light_Box">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="Remove_Light">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Ctrl+Shift+L</string>
           </property>
           <property name="text">
            <string>Remove Light</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+Shift+L</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="Modify_Light">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>26</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>26</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Ctrl+E</string>
           </property>
           <property name="text">
            <string>Modify Light</string>
           </property>
           <property name="shortcut">
            <string>Ctrl+E</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="GroupBox_Geomtery">
       <property name="title">
        <string>Modify Model Geometry</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="3" column="0">
           <widget class="QPushButton" name="delete_Axes_PushButton">
            <property name="text">
             <string>Delete Model Axes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="Delete_Ruler_PushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Ctrl+D</string>
            </property>
            <property name="text">
             <string>Delete Ruler</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+D</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="Axes_PushButton">
            <property name="text">
             <string>View Model Axes</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="Ruler_PushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Shift+R</string>
            </property>
            <property name="text">
             <string>Insert Ruler</string>
            </property>
            <property name="shortcut">
             <string>Shift+R</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="View_Statistics_of_Model">
            <property name="text">
             <string>View Model Statistics</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string> Apply Filter to Model</string>
       </property>
       <widget class="QCheckBox" name="Clipper_Filter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>79</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter Clip</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="Shrink_Filter">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>79</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter Shrink</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="Colors_groupBox">
       <property name="title">
        <string>       Modify Colours</string>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>151</width>
          <height>161</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="Change_Back_Ground_Color">
           <property name="text">
            <string>change Background color</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="Change_Object_Color">
           <property name="text">
            <string>change  Model color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="Change_Back_FaceColor">
           <property name="text">
            <string>Change backFace color</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="Reset_Color">
           <property name="text">
            <string>Reset Colours</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="Shapes_groupBox">
       <property name="title">
        <string>               Shapes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0" rowspan="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QPushButton" name="cube">
            <property name="text">
             <string>cube</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="cone">
            <property name="text">
             <string>Cone</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="sphere">
            <property name="text">
             <string>sphere</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionHelp"/>
    <addaction name="actionPrint"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionHelp"/>
   <addaction name="actionPrint"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>Icons/fileopen.png</normaloff>Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Icons/filesave.png</normaloff>Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>Icons/help.png</normaloff>Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>Icons/print.png</normaloff>Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>qvtkopenglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
